services/dashboard/dashboards/parsers.go

```diff
@@ -130,11 +129,6 @@ func (widget *DashboardWidget) FromMap(widgetMap map[string]any) error {
 	} else {
 		widget.Series = series
 	}
-	if metrics, err := helper.ParseFromMaps[widgets.MetricInfo](widgetMap["metrics"].([]any)); err != nil {
-		return err
-	} else {
-		widget.Metrics = metrics
-	}
```

Comment:
Not related to this ticket, but I break the Terraform build with my previous PR 😬